### PR TITLE
Issue template

### PR DESCRIPTION
**Pull request by stevepiercy**
[Fixes: #1172](https://github.com/plone/documentation/issues/1172)

Changes proposed in this pull request:
- Add new issue template and chooser

### CODE REVIEW

**Review by nileshgulia1 — APPROVED** [review]

**Comment by nileshgulia1** [comment]
@jenkins-plone-org please run jobs

File: .github/ISSUE_TEMPLATE/new-issue-form.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Report an Issue
+description: Report an issue with Plone Documentation
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this issue report!
```
Comment (ksuess) [inline .github/ISSUE_TEMPLATE/new-issue-form.yml:7]: this is nice.

**Review by ksuess — COMMENTED** [review]
Well, I still like the request to the issuers to optionally click a category of the report, as @stevepiercy  proposed. It's optional, but this info lowers the barrier to contributors to pick the issue and do something. plone/documentation is a side side project for many of us and we have focus to lots of other projects. So if I see an issue that I understand what he/she is talking about: i am interested and go and enhance the documentation.

this one:


> I am reporting: (radios)
> 
> 1. A correction in content is needed (typographical error, misspelling, broken link, outdated or incorrect content)
> 2. Missing content
> 3. Automation, build, testing, or deployment issue
> 4. Suggestion to enhance or improve the documentation
> 5. Other

**Comment by stevepiercy** [comment]
I found that radio buttons are not supported, but select menus (dropdown) are. meh.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#dropdown

Selects can be optional or required.

I feel that collecting this information is a solution in search of a problem.

- As someone submitting an issue, it slows me down. I dislike select menus because you cannot see all the options until you click on it. And I often describe the issue type in the Description anyway.
- As someone triaging an issue, I would find the information from the select response to be frequently redundant. The Description often describes the issue type anyway.

**Comment by ksuess** [comment]
> I found that radio buttons are not supported, but select menus (dropdown) are. meh.
> 
> https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#dropdown
> 
> Selects can be optional or required.
> 
> I feel that collecting this information is a solution in search of a problem.
> 
>     * As someone submitting an issue, it slows me down. I dislike select menus because you cannot see all the options until you click on it. And I often describe the issue type in the Description anyway.
> 
>     * As someone triaging an issue, I would find the information from the select response to be frequently redundant. The Description often describes the issue type anyway.

Convinced. Skip the issue category.

**Review by ksuess — APPROVED** [review]

**Comment by stevepiercy** [comment]
@jenkins-plone-org please run jobs

**Comment by stevepiercy** [comment]
I have no clue what Jenkins is supposed to do, and I am not going to wait around to find out. Merging!

**Comment by ksuess** [comment]
Realised just now: Why not merge in branch '5.2' (main branch)?

**Comment by stevepiercy** [comment]
I don't want to touch the `5.2` branch and take on the responsibility for mucking it up. I would prefer to set the default branch of this repo to `6-dev` so that it uses the new issue template and chooser.

I would like to discuss that with the team before changing the setting for the default branch. @polyester @tkimnguyen @svx @spereverde @ksuess what do you all think?

**Pull request merged**